templates/todo/projects/python-mongo/.repo/terraform/repo.yaml

```diff
@@ -109,6 +109,14 @@ repo:
         - .github/**/*
         - .devcontainer/**/*
         - "infra/**/*"
+        - .azdo/**/*
```

Comment:
Can you ignore only `.azdo/pipelines/*/azure-dev.yml` ?   So we can still have the README.md copied for all?  
Or is there a reason why we want to exclude everything from .azdo/ folder?

Reply:
good find @vhvb1989, updated it